assets/js/search.js

```diff
@@ -114,9 +114,13 @@ function search (nodes, value) {
   $oldContent.hide()
   $content.append($results)
 
-  function closeResults (event) {
-    if (event.metaKey) {
-      return
+  function closeResults (e) {
+    var event = e || window.event
+    if (typeof event !== 'undefined' && typeof event === 'object' && event !== null) {
```

Comment:
Honestly, this looks like a smell. Don't  we only need `typeof event === 'object'`? Can it have a type of an object and still be null?